utils/debug/PointCloudDebug.js

```diff
@@ -10,39 +10,36 @@ function isInHierarchy(elt, hierarchyNode) {
 
 export default {
     initTools(view, layer, datUi) {
+        const update = () => view.notifyChange(true);
         layer.debugUI = datUi.addFolder(`${layer.id}`);
 
-        layer.debugUI.add(layer, 'sseThreshold').name('SSE threshold')
-            .onChange(() => view.notifyChange(true));
-        layer.debugUI.add(layer, 'octreeDepthLimit', -1, 20).name('Depth limit')
-            .onChange(() => view.notifyChange(true));
-        layer.debugUI.add(layer, 'pointBudget', 1, 15000000).name('Max point count')
-            .onChange(() => view.notifyChange(true));
+        layer.debugUI.add(layer, 'sseThreshold').name('SSE threshold').onChange(update);
+        layer.debugUI.add(layer, 'octreeDepthLimit', -1, 20).name('Depth limit').onChange(update);
+        layer.debugUI.add(layer, 'pointBudget', 1, 15000000).name('Max point count').onChange(update);
         layer.debugUI.add(layer.object3d.position, 'z', -50, 50).name('Z translation').onChange(() => {
             layer.object3d.updateMatrixWorld();
             view.notifyChange(true);
         });
-        layer.debugUI.add(layer, 'pointSize', 0, 15).name('Point Size')
-            .onChange(() => view.notifyChange(true));
-        layer.debugUI.add(layer, 'opacity', 0, 1).name('Opacity')
-            .onChange(() => view.notifyChange(true));
 
         layer.dbgEnableStickyNode = false;
         layer.dbgStickyNode = '';
         layer.dbgDisplayChildren = true;
         layer.dbgDisplayParents = true;
         layer.dbgDisplaybbox = false;
 
+        var styleUI = layer.debugUI.addFolder('Styling');
+        styleUI.add(layer, 'opacity', 0, 1).name('Layer Opacity').onChange(update);
+        styleUI.add(layer, 'pointSize', 0, 15).name('Point Size').onChange(update);
+        styleUI.add(layer.material, 'visible').name('Display Points').onChange(update);
```

Comment:
The same checkbox is called `Visible` for other layers. It makes sense because it also hides the bounding box btw. 

Also for me it's not a matter of styling but more something useful to hide temporarily a layer. Can you move this on the parent folder?